packages/terra-time-input/src/TimeInput.jsx

```diff
@@ -155,18 +155,8 @@ class TimeInput extends React.Component {
     let meridiem;
 
     if (TimeUtil.getVariantFromLocale(props) === TimeUtil.FORMAT_12_HOUR) {
```

Comment:
Instead of having two separate if block we can alter this condition like this.

if (this.props.variant === TimeUtil.FORMAT_12_HOUR) {
   if (TimeUtil.getVariantFromLocale(props) === TimeUtil.FORMAT_24_HOUR)) {
    ......................;
   }
  else {
    .....................;
  }
};